data/change-streams/unified/change-streams.json

```diff
@@ -109,6 +120,264 @@
               ]
             }
           }
+        },
+        {
```

Comment:
Looking at the test failures, I think the format of this file may be corrupted. It looks like these new tests are in the "operations" array of the previous test.
```
=== RUN   TestUnifiedSpec/change-streams/unified/change-streams.json/Test_array_truncation
    unified_spec_runner.go:130: error running operation "" at index 4: execution failed: unrecognized entity operation ""
```